contracts/registries/YRegistryV1.sol

```diff
@@ -0,0 +1,219 @@
+//SPDX-License-Identifier: Unlicense
+pragma solidity ^0.6.8;
+
+import "@openzeppelinV3/contracts/math/SafeMath.sol";
+import "@openzeppelinV3/contracts/utils/Address.sol";
+import "@openzeppelinV3/contracts/utils/EnumerableSet.sol";
+
+import "../../interfaces/yearn/IController.sol";
+import "../../interfaces/yearn/IStrategy.sol";
+import "../../interfaces/yearn/IVault.sol";
+import "../../interfaces/yearn/IWrappedVault.sol";
+
+contract YRegistryV1 {
+  using Address for address;
+  using SafeMath for uint256;
+  using EnumerableSet for EnumerableSet.AddressSet;
+
+  address public governance;
+  address public owner;
+  
+  address public pendingGovernance;
+  address public pendingOwner;
+
+  EnumerableSet.AddressSet private vaults;
+  EnumerableSet.AddressSet private controllers;
+
+  mapping(address => address) private wrappedVaults;
+
+  mapping(address => bool) public isDelegatedVault;
+
+  constructor() public {
+    owner = msg.sender;
+    governance = msg.sender;
+  }
+  
+  function isYRegistry() external pure returns (bool) {
+    return true;
+  }
```

Comment:
What is this for?

Reply:
Just a check. Can be removed since it's used in any other contract.